docs/docs/notification-center/iframe-embed.md

```diff
@@ -1,7 +1,7 @@
 # iFrame Embed
 
-If you are using an unsupported (yet) client framework, you can use our embed script, this will generate the notification center inside an iframe.
-You can find the embed code in the `Settings` page within the Admin Panel. It will look similar to this:
+If you are using an (currently) unsupported client framework, you can use our embed script. This will generate the notification center inside an iframe.
```

Comment:
```suggestion
If you are using n (currently) unsupported client framework, you can use our embedded script. This will generate the notification center inside an iframe.
```

Reply:
This doesn't make sense

Reply:
Sorry, fixed the mistyping.